File: examples/fib.py
import time

def fib(n):
    if n <= 1:
        return n
    else:
        return fib(n - 1) + fib(n - 2)


start = time.time()
print(fib(30))
end = time.time()

print("Elapsed time:", (end - start))

File: examples/fib.js
function fib(n) {
  if (n <= 1) {
    return n;
  } else {
    return fib(n - 1) + fib(n - 2);
  }
};

const start = performance.now();
console.log(fib(30));
const end = performance.now();

console.log("Elapsed time:", (end - start) / 1000);
